pubpub/pubpub
# Integrate the Pub Connections edge proposal API and edge editor

This PR lets you add Pub connections by PubPub URL or DOI, and edit edges that resolve with an `ExternalPublication`:
![Screen Recording 2020-07-10 at 9 54 51 AM 2020-07-10 10_08_24](https://user-images.githubusercontent.com/2208769/87163405-4e6e7980-c295-11ea-8c9d-10935100313b.gif)

_Test plan:_
- Enter a PubPub url like https://cursor.pubpub.org/pub/uv2c6nd4/merge/2/1 and ensure you can create an edge
- Enter a DOI like `10.1016/j.trd.2010.12.004` and ensure you can create an edge, as well as edit its metadata.
- Enter any old url and ensure that you can create an edge with it, even if we don't resolve a useful proposal.
